cobra/io/sbml.py

```diff
@@ -770,7 +772,7 @@ def process_association(ass):
     else:
         # parse deprecated subsystems on reactions
         groups_dict = {}
-        for cobra_reaction in cobra_model.reactions:
+        for cobra_reaction in cobra_model.reactions: # TODO - reaction subsystems are empty, so perhaps they should be filled in or reactions should have groups
```

Comment:
The SUBSYSTEMS are parsed in groups. Not sure what the TODO means here. Could you explain?

Reply:
Each reaction has the field subsystem.
After adding subsystems to groups, the ".subsystem" field in each reaction remains empty. I think the subsystem field should link to the groups and/or each reaction should have a ".groups" field.

Reply:
There is some work in progress on adding compatibility for SBML groups indeed.